### PR TITLE
Add an office hours page.

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

File: officehours.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Office Hours
+sidebar_link: true
+sidebar_sort_order: 250
+---
+
+Boston Python holds a weekly office hour, Mondays at noon.
```
Comment (bhrutledge) [inline officehours.md:7]:
```suggestion
Boston Python holds a weekly Zoom office hour on Mondays at noon, sponsored by [edX](https://edx.org)
```

File: officehours.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+This is an unstructured time to drop in to ask questions or just chat. There is no set topic, anything goes: Python, Boston Python, specialized technical topics, adjacent technologies, it's all good.  For now, this is an experiment. We may change the structure, pace, or timing as we see what happens.
+
+The office hour happens on [Zoom](ZOOM LINK TO COME), sponsored by [edX](https://edx.org).
+
+You do not have to sign up ahead of time, just drop in.  See you there!
```
Comment (bhrutledge) [inline officehours.md:13]:
```suggestion
You do not have to sign up ahead of time, just drop in at <https://{{ zoom_url }}>.  See you there!
```

**Review by bhrutledge — COMMENTED** [review]
Thanks for writing this up. I think an explicit URL might be more obvious and less surprising. That said, any hesitation about posting this on a public, crawl-able page?

**Comment by nedbat** [comment]
I could leave the Zoom link out, and tell people to find the meetup.com event page instead.  Meetup only shows the link to members, I think?

File: officehours.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+sidebar_sort_order: 250
+---
+
+Boston Python holds a weekly office hour, Mondays at noon.
+
+This is an unstructured time to drop in to ask questions or just chat. There is no set topic, anything goes: Python, Boston Python, specialized technical topics, adjacent technologies, it's all good.  For now, this is an experiment. We may change the structure, pace, or timing as we see what happens.
+
+The office hour happens on [Zoom](https://edx-org.zoom.us/j/85746000546?pwd=N3RyR2VsbGFMTFdPUUUva3VhTlFHdz09), sponsored by [edX](https://edx.org).
+
+You do not have to sign up ahead of time, just drop in.  See you there!
```
Comment (bhrutledge) [inline officehours.md:13]:
```suggestion
You do not have to sign up ahead of time; just [drop in on Zoom](https://edx-org.zoom.us/j/85746000546?pwd=N3RyR2VsbGFMTFdPUUUva3VhTlFHdz09).  See you there!
```

**Review by bhrutledge — COMMENTED** [review]
Some more word-smithing/bike-shedding

File: officehours.md (hunk 3 above)
Comment (bhrutledge) [reply officehours.md:13]: @nedbat:

> I could leave the Zoom link out, and tell people to find the meetup.com event page instead. Meetup only shows the link to members, I think?

Don't know; that would add some friction to people joining. I don't feel strongly; just a thought.

**Comment by nedbat** [comment]
@bhrutledge thanks, take another look.  I moved the Zoom link to the meetup page just to be cautious.

**Review by bhrutledge — APPROVED** [review]
Nice!

**Pull request merged**